Woof/Woof/View/OwnerFlow/OwnerDetailedWalkingView/InformationCellView.swift

```diff
@@ -0,0 +1,71 @@
+import SwiftUI
+
+/// Represents an information cell with provided label, text, and icon.
+/// Can be displayed in one line or presents `text` under the `icon` and `label`.
+struct InformationCellView: View {
+    /** Initializes an information cell view with provided parameters.
+
+     - Parameters:
+     - label: The label text for the information.
+     - text: The text to display in the cell.
+     - imageName: The name of the system image to display.
+     - showedInOneLine: Indicates whether the information should be presented in one line or not, by default is `true`.
+      */
+    init(label: String, text: String, imageName: String, showedInOneLine: Bool = true) {
+        self.label = label
+        self.text = text
+        self.imageName = imageName
+        self.showedInOneLine = showedInOneLine
+    }
+
+    var body: some View {
+        VStack(
+            alignment: .leading,
+            spacing: AppStyle.UIElementConstant.minPadding
+        ) {
+            if showedInOneLine {
+                HStack {
+                    Label(
+                        label,
+                        systemImage: imageName
+                    )
+                    Text(text)
+                        .bold()
+                }
+            } else {
+                Label(
+                    label,
+                    systemImage: imageName
+                )
+                Text(text)
+                    .bold()
+            }
+
+            Divider()
+        }
```

Comment:
If I correctly get your idea, we should remove high-level VStack.
If we remove the high-level VStack and try to use our cell in HStack as a parent container it will be aligned in HStack order, and the divider will be at the trailing border. View builder arranges and aligns views based on the parent view specification if there are no provided restrictions in the child view.

Reply:
<img width="1142" alt="Screenshot 2024-05-08 at 09 27 58" src="https://github.com/IronFoundation/iOS-Woof/assets/32869814/aa8629c0-2493-44ce-af34-5c4c1c1f76d7">

Reply:
Yes, you're right!